frontend/lib/src/components/widgets/TextArea/TextArea.tsx

```diff
@@ -188,7 +188,7 @@ class TextArea extends React.PureComponent<Props, State> {
     )
 
     return (
-      <div className="stTextArea" style={style}>
+      <div className="stTextArea" data-testid="stTextArea" style={style}>
```

Comment:
nit: moving test IDs might actually break some apps since people started to rely on `data-testid` for custom CSS styling. But it is probably not something we need to care about since we are not advertising `data-testid` as a reliable way for custom styling.

Reply:
Oh didn't think about that, will avoid moving the existing